lightning/src/ln/channelmanager.rs

```diff
@@ -4681,6 +4683,8 @@ impl<'a, Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref>
 				log_error!(args.logger, "Missing ChannelMonitor for channel {} needed by ChannelManager.", log_bytes!(channel.channel_id()));
 				log_error!(args.logger, " The chain::Watch API *requires* that monitors are persisted durably before returning,");
 				log_error!(args.logger, " client applications must ensure that ChannelMonitor data is always available and the latest to avoid funds loss!");
+				log_error!(args.logger, " Without the ChannelMonitor we cannot continue without risking funds.");
+				log_error!(args.logger, " Please ensure the chain::Watch API requirements are met file a bug report at https://github.com/rust-bitcoin/rust-lightning");
```

Comment:
Just need a "and" between "Please ensure the chain::Watch API requirements are met and file a bug report at https://github.com/rust-bitcoin/rust-lightning" ?